src/services/swapServices/index.ts

```diff
@@ -9,6 +10,11 @@ import { formatDecimalsFromToken } from "../../app/util/helper";
 
 const { parents } = useGetNetwork();
 
+const checkIfExactError = (errorValue: string) => {
+  if (errorValue === t("swapPage.palletSlippageError")) return true;
+  return false;
```

Comment:
U can just add it here 
return errorValue === t("swapPage.palletSlippageError")